cyclops/utils/file.py

```diff
@@ -176,7 +183,7 @@ def load_dataframe(
     file_format: str = "parquet",
     log: bool = True,
 ) -> pd.DataFrame:
-    """Load file to a pandas.DataFrame object.
+    """Load file to a pandas.DataFrame or dask.DataFrame object.
```

Comment:
Seems like you forgot to update the docstring and return type hints here.

Also, should it be "Yields" instead of "Returns" in the docstring?